### DISCUSSION TITLE
表单setFieldsValue后无法通过getFieldsValue获取表单最新数据

### DISCUSSION THREAD

**Discussion by auqf** [vbenjs/vue-vben-admin#3746] [Q&A]
demo如下：
      labelForm.setFieldsValue(newDict)
      console.log(486, labelForm.getFieldsValue())
![image](https://github.com/vbenjs/vue-vben-admin/assets/8895540/3b64c094-db45-4bdd-a77e-10c153ed140e)

**Comment by lackoxy** [comment]
labelForm.setFieldsValue(newDict) 前加一个await
await labelForm.setFieldsValue(newDict)
